### PR TITLE
Update for Caustic

### PR DESCRIPTION
**Pull request by Waterpicker**
Signed-off-by: waterpicker waterpickerenternity@gmail.com

### CODE REVIEW

File: src/main/java/com/flowpowered/engine/render/FlowRenderer.java [hunk 1]
```diff
@@ -68,21 +71,22 @@
...
 
 /**
- * The default renderer. Support OpenGL 2.1 and 3.2. Can render fully textured models with normal and specular mapping, ambient occlusion (SSAO), shadow mapping, Phong shading, motion blur and edge
- * detection anti-aliasing. The default OpenGL version is 3.2.
+ *
  */
 public class FlowRenderer implements Renderer {
-    private final String WINDOW_TITLE = "Flow Engine";
-    private final DateFormat SCREENSHOT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss");
+    private static final String WINDOW_TITLE = "Spoutcraft";
```
Comment (DDoS) [inline src/main/java/com/flowpowered/engine/render/FlowRenderer.java:88]: Might want to make this Flow Engine again

File: src/main/java/com/flowpowered/engine/render/FlowRenderer.java [hunk 2]
```diff
@@ -239,49 +249,39 @@ private void initMaterials() {
...
         addTransparentModel(model1);
         final Model model2 = model1.getInstance();
         model2.setPosition(new Vector3f(0, 22, 6));
-        model2.getUniforms().add(new Vector4Uniform("modelColor", new Vector4f(0, 0, 1, 0.7)));
+        model2.getUniforms().add(new Vector4Uniform("modelColor", new Vector4f(0, 1, 1, 0.7)));
         addTransparentModel(model2);
     }
 
-    private void addHUD() {
+    private void addFPSMonitor() {
```
Comment (DDoS) [inline src/main/java/com/flowpowered/engine/render/FlowRenderer.java:267]: For this method you should only replicate the changes made in Client, as the Flow version has some added stuff that you just removed.

File: src/main/java/com/flowpowered/engine/render/FlowRenderer.java [hunk 3]
```diff
@@ -311,39 +311,28 @@ public void render() {
...
         for (Model model : renderModelsNode.getModels()) {
             model.getUniforms().getMatrix4("previousModelMatrix").set(model.getMatrix());
         }
-        for (Model model : renderTransparentModelsNode.getModels()) {
-            model.getUniforms().getMatrix4("previousModelMatrix").set(model.getMatrix());
-        }
     }
 
-    private void updateHUD() {
+    private void updateFPSMonitor() {
```
Comment (DDoS) [inline src/main/java/com/flowpowered/engine/render/FlowRenderer.java:331]: Same thing here.

**Comment by kitskub** [comment]
Pulled in: f7a4686a734dd032b2dd62054d44112d84275743

**Pull request closed without merging**